src/H5TSatomic.c

```diff
@@ -162,6 +162,57 @@ H5TS_atomic_destroy_uint(H5TS_atomic_uint_t *obj)
     FUNC_LEAVE_NOAPI_VOID_NAMECHECK_ONLY
 } /* end H5TS_atomic_destroy_uint() */
 
+/*--------------------------------------------------------------------------
+ * Function:    H5TS_atomic_init_voidp
+ *
+ * Purpose:     Initializes an atomic 'void *' variable object with a value.
+ *
+ * Note:        Per the C11 standard, this function is not atomic and
+ *              concurrent execution from multiple threads is a data race.
+ *
+ * Return:      None
+ *
+ *--------------------------------------------------------------------------
+ */
+void
+H5TS_atomic_init_voidp(H5TS_atomic_voidp_t *obj, void *desired)
+{
+    FUNC_ENTER_NOAPI_NAMECHECK_ONLY
+
+        /* Initialize mutex that protects the "atomic" value */
```

Comment:
Why is clang-format messing up the indenting?

Reply:
> Why is clang-format messing up the indenting?

I'll see what I can do about it

Reply:
I spent some time looking at the clang-format directives (https://clang.llvm.org/docs/ClangFormatStyleOptions.html) but it wasn't clear what might be controlling this mis-format.  So, I decided to just try removing the `(void)` cast, and since that doesn't generate warnings, I went with that solution.